smem: add page (#8163)
tldr-pages/tldr: pages/linux/smem.md

# smem

> > Show memory usage for programs.

Reviewer: ```suggestion
> Print memory usage for programs.
```

> More information: <https://manned.org/smem>.

> - Print memory usage for current processes:

Reviewer: ```suggestion
- Print memory usage for all current processes:
```

It's more explicit.

`smem`

> - Print memory usage for current processes for a every user on a system:

Reviewer: ```suggestion
- Print memory usage for all current processes for an every user on a system:
```

Reviewer 2: I think adding `a/an` does not sound right. Although, not an expert but in my opinion, I guess it should be:
```suggestion
- Print memory usage for all current processes for every user on the system:
```

Reviewer 3: @CleanMachine1 wdyt about the suggestion of mfrw?

Author: I dont think it is all current processes therefore don't think these changes are necessary, to be honest I don't know much about the command.

Reviewer 3: ok, let's merge then.

`smem --users`

> - Print memory usage for current processes for a specified user:

Reviewer: ```suggestion
- Print memory usage for all current processes for a specific user:
```

Author: From what I can tell, it is not all processes

> `smem --userfilter {{username}}`

Reviewer: ```suggestion
`smem --userfilter {{username}}`
```

It is not clear what regex type is supported from this code sample. What about to note about it in the description smth like (`{{regex-type}}` is used)?

Reviewer: Sorry.

Author: No problem!
